packages/api/src/types.ts

```diff
@@ -127,6 +141,16 @@ export type FeedConfig = {
   feeds: FeedInfoRouterConfigMap
 }
 
+export type ExtendedFeedConfig = {
+  address: string
+  blockExplorer: string
+  color: string
+  name: string
+  chain: string
+  pollingPeriod: number
+  feeds: FeedInfoRouterConfigMap
+}
+
```

Comment:
in collation to previous comment, adding providerURL to both config file and here, could be a future option just to have in mind